src/main/java/com/google/sps/servlets/DatabaseLoaderServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+
+import com.google.sps.data.DatabaseHandler;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+
+/**
+ * When user goes to /load-database, he needs to wait for the database to get loaded.
+ * Will parse the CSV file to create fictive book stocks.
+ * 
+ * Can also be loaded as a cron job from here:
+ * https://pantheon.corp.google.com/appengine/cronjobs?project=alphabooks-step-2020
+ */
+
+@WebServlet("/load-database")
+public class DatabaseLoaderServlet extends HttpServlet {
+
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    File csvFile;
+    try {
+      csvFile = new File(this.getClass().getResource("/availableBooks.csv").toURI());
+    } catch (URISyntaxException e) {
+      System.out.println("File not found!");
```

Comment:
nit: this seems redundant. given the error and the stacktrace.

Reply:
Fixed